ios/RCTWebRTC/WebRTCModule+Daily.m

```diff
@@ -153,7 +153,7 @@ - (void)applyAudioMode:(NSString *)audioMode toSession:(RTCAudioSession *)audioS
   // other apps, which allows this app to stay alive in the backgrounnd during
   // a call (assuming it has the voip background mode set).
   AVAudioSessionCategoryOptions categoryOptions = (AVAudioSessionCategoryOptionAllowBluetooth |
-                                                   AVAudioSessionCategoryOptionDuckOthers);
+                                                   AVAudioSessionCategoryOptionMixWithOthers);
```

Comment:
Sorry I never got around to testing this, but if it sounds reasonable to you I trust you. Seems like a better default to "duck a bit due to the WebRTC bug" than to "duck a bit due to the WebRTC bug and then duck _even more_ on top of that"